src/main/java/task/Task.java

```diff
@@ -0,0 +1,25 @@
+package task;
+
+public class Task {
+    protected String name;
+    protected boolean isDone;
+
+    public Task(String name) {
+        this.name = name;
+        isDone = false;
+    }
+
+    public void markAsDone() {
+        isDone = true;
+    }
+
+    protected String getStatusIcon() {
+        //return tick or X symbols
+        return (isDone ? "\u2713" : "\u2718");
```

Comment:
Maybe renaming these symbols to a constant variable will make the code easier to understand at a glance (since developers might not know what that unicode means), it will also make it easier to edit them if required